layouts/partials/modules/products.html

```diff
@@ -47,6 +47,9 @@ <h2>{{$.title}}</h2>
         {{ warnf "Could not find color %s" .}}
         {{ end }}
         {{ end }}
+        {{ if eq $index 5}}
+        <a href="{{$link}}" class="color-dot--see-more">+</a>
+        {{ end }}
```

Comment:
This is a pretty nice solution with minimal code changes! However, can we make this show the `+` button only if there are 7 colors or over? So 6 colors are still shown normally, but 7 colors would render the first 5 colors and then the `+` button.

For this you probably want to create the pretty complicated double `apply` a separate `$colors` variable or something? I.e. `{{ $colors := (apply (apply .Params.variants "index" "." "options") "index" "." $coloroption) | uniq }}`.

*Warning for very picky comment*: If we follow BEM naming to the dot, this class should probably be named `color-dots__see-more`.

Reply:
Ah, thanks! This is what I wanted to do but I didn't figure out how to create separate variable. How about now?
Also changed class name to color-dots__see-more